src/preprocess.rkt

```diff
@@ -71,13 +58,28 @@
   (define sort-instructions
     (for/list ([component (in-list components)]
                #:when (> (length component) 1))
-      (cons 'sort component)))
+      (cons 'sort component)))      
+  (define preprocessing (append abs-instructions sort-instructions))
+  (define alternative (make-alt-preprocessing initial preprocessing))
+   ;; Absolute value should happen before sorting
+  (define simplified
+    (cons
+     ;; We excluded the first element of `initials` above so that we can add it
```

Comment:
Why move this code around in the file? Is there a point?

Reply:
clarity, but jackson's PR has made it irrelevant